<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsMenu</class>
 <widget class="QMainWindow" name="ToolsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_flip_vertically">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip vertically</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_flip_horizontally">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip horizontally</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_image">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose an image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_colors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nยบ Colors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_luminance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Luminance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quantize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copy">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy to edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_histogram">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_brightness">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_brightness">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_contrast">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set contrast</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_contrast">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_negative">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_histogram_match">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram match</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equalize_histogram">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Equalize histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_zoomout">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom out</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_zoomout_x">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_zoomout">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> X                     Y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_zoomout_y">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_zoomin">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom in</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zoomin">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom in 2x2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rotate90p">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+90ยบ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rotate90n">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-90ยบ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_convolution">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Convolve</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_a">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_b">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_c">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_d">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_e">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_f">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_g">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_h">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_kernel_i">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-99.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kernel_gaussian">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kernel_laplacian">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Laplacian</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kernel_hpass">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>H. Pass</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kernel_prewitthx">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt Hx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kernel_prewitthy">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt Hy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kernel_sobelhx">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>70</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Hx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kernel_sobelhy">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>100</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sobel Hy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_graybg">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Background</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_contrast">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>150</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <double>255.990000000000009</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_brightness">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-255</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_colors">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_credit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>By Gabriel Madeira</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
